docs/integrations/aws.md

```diff
@@ -40,43 +41,84 @@ Provide a **Handle** for the integration.
 
 This handle should be meaningful and must be unique for all integrations in the tenant (including any org-level integrations).  Click **Next**.
 
+## Step 3: Setup Access to Your Organization Management Account
+
+In this step, Pipes uses:
+
+- **`External ID`** ensures secure access between Pipes and AWS accounts. Refer to AWS documentation on [Access to AWS accounts owned by third parties](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html) for further information.
+- **`IAM Role with Cross Account Trust`** allows Turbot Pipes to access resources across accounts. For additional context, see AWS's guide on [Cross-account resource access in IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies-cross-account-resource-access.html).
+
+![Cross Account Trust](/images/docs/pipes/integrations/aws-setup-org-access.png)
+
+### Cross Account Trust
+
+The IAM role must grant cross-account access for the Turbot Pipes main AWS account to assume into your AWS account.
+
+- Turbot Pipes customers, you must allow the Turbot Pipes AWS Account ID: `316881668097`
+
+### External ID Considerations
+
+The External ID is a security measure that helps prevent unauthorized access. It consists of two parts:
+- Your organization ID `o_cugqnt0sb890gkker123` (automatically added)
+- A random 8-character string (you can customize this)
```

Comment:
@karanpopat what are the valid chars? Is it alphanumeric?

Reply:
It's supposed to be alphanumeric. Updated it